lib/team_api/joiner.rb

```diff
@@ -121,15 +132,19 @@ def team_members
     # - Strings that are GitHub usernames
     # - Hashes that contain an 'email' property
     # - Hashes that contain a 'github' property
-    def join_team_list(team_list)
-      (team_list || []).map! do |reference|
+    def join_team_list(team_list, errors)
+      list_team = (team_list || [])
+      list_team.map! do |reference|
```

Comment:
Why hoist the `(team_list || [])` expression and the `end.concat` out from where they were?